tools/ImportTool.Shared/SplitBlobsCommand.cs

```diff
@@ -0,0 +1,151 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Extensions.Logging;
+
+namespace ImportTool.Shared
+{
+    public static class SplitBlobsCommand
+    {
+        private static ILogger _logger = GetLogger();
+
+        private static ILogger GetLogger()
+        {
+            using (var factory = LoggerFactory.Create(builder => builder.AddConsole()))
+            {
+                return factory.CreateLogger(typeof(Program).FullName);
+            }
+        }
+
+        public static async Task Split(
+            string account,
+            string key,
+            string prefix,
+            long size,
+            long blockSize,
+            int maxConcurrentFile,
+            int maxSpliterPerBlob,
+            int maxUploaderPerBlob)
+        {
+            string storageConnectionString = GetConnectionStringFromAccountAndKey(account, key);
+
+            CloudStorageAccount storageAccount;
+            try
+            {
+                if (CloudStorageAccount.TryParse(storageConnectionString, out storageAccount))
+                {
+                    _logger.LogInformation("Start to split files in storage {0} with prefix {1}", account, prefix);
+
+                    CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                    int count = await SplitBlob(
+                        cloudBlobClient,
+                        prefix,
+                        size,
+                        blockSize,
+                        maxConcurrentFile,
+                        maxSpliterPerBlob,
+                        maxUploaderPerBlob);
+
+                    _logger.LogInformation("Split {0} files in total", count);
+                }
+                else
+                {
+                    throw new StorageException(
+                        "An invalid connection string.");
```

Comment:
as it's a separated project with fhir-server, so it may not use reosurce-designer in fhir-server.
currently we only have this one place to throw exception with error message , maybe create a self-used Resource file when error is more.